src/js/messaging/utils/validations.js

```diff
@@ -0,0 +1,20 @@
+export function validateNumAttachments(files, maxAttachments) {
+  return files.length > maxAttachments;
+}
+
+export function validateFileSize(files, max) {
+  return !!files.find((file) => { return file.size > max; });
+}
+
+export function validateTotalFileSize(files, max) {
+  // Get sizes for each file.
+  const sizes = files.map((f) => {
+    return f.size;
+  });
+
+  const total = sizes.reduce((a, b) => {
+    return a + b;
```

Comment:
Can this just be one pass? Just a reduce that returns `a + b.size` and an initial value of 0. Less readable, but a comment should be clear enough.